api/config_keys.py

```diff
@@ -0,0 +1,46 @@
+from pydantic import BaseModel
+from typing import Optional
+
+
+class APP(BaseModel):
+    VideoPath: Optional[str]
+    Host: Optional[str]
+    Port: Optional[str] 
```

Comment:
Does it make sense to allow changing these as part of an endpoint request?

Reply:
Hm. This should be set in configuration which is used by the dashboard to connect to the api, so I think this is a wrong place for having it.